### ISSUE TITLE
React 0.14 compatibility

### ISSUE THREAD

**Issue by rayshan** [eiriklv/react-masonry-component#4]
Hi Eirik, thanks for building this. When using with React 0.14 RC1:

If using `React.render()`, the app works fine, but I get:

```
Warning: React.render is deprecated. Please use ReactDOM.render from require('react-dom') instead.
```

```
Warning: React.findDOMNode is deprecated. Please use ReactDOM.findDOMNode from require('react-dom') instead.
```

If importing ReactDOM and using `ReactDOM.render()`, the app does not work, I get:

```
Uncaught (in promise) Error: Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```

**Comment by eiriklv** [comment]
Thanks, will take a look :-)

**Comment by rayshan** [comment]
I forgot to mention I'm using webpack.

The `Invariant Violation` can be fixed by either including React as an npm dependency and setting the version to `>=0.13||>=0.14-alpha` under npm 2, or using npm 3. Otherwise webpack bundles multiple copies of React.

The remaining issue is `Warning: React.findDOMNode is deprecated.` It should be an easy fix within this library.

**Comment by fskinner** [comment]
:+1: having the same issue. just did a Find All on my project and there no `findDOMNode` on my source code tho

**Comment by eiriklv** [comment]
Feel free to submit a PR for this :-) Will have to release a new major version since this will break backwards compatibility.

**Comment by WellDone2094** [comment]
to solve this warning you just need to use ReactDOM.render to render ur root component in the dom
```
ReactDOM.render(
  <MyRootComponent/>,
  document.getElementById('container'));
);
```

**Comment by WellDone2094** [comment]
to solve this you need to use ReactDOM.render to render your root element,
```
ReactDOM.render(
       (<Body/>),
       document.getElementById('container')
);
```

**Comment by Hancoson** [comment]
need to use <script src="../build/react-dom.js"></script>
ReactDOM.render()

**Comment by afram** [comment]
Fixed from v2.0.0
